arangodb-net-standard.Test/IndexApi/IndexApiClientTest.cs

```diff
@@ -61,25 +60,6 @@ public async Task PostIndexAsync_ShouldSucceed()
             Assert.NotNull(indexId);
```

Comment:
I think we should update the test to retrieve the index information and confirm that `Unique` is indeed true.

Justification: the serializer will receive a `PostIndexBody` but should still serialize the properties of the sub class `PostPersistentIndexBody`. It works fine with Json.NET but it would be good to have coverage on this behavior in case we change the default serializer in future.